ci-operator/step-registry/cucushift/installer/check/azure/confidential/cucushift-installer-check-azure-confidential-commands.sh

```diff
@@ -123,58 +111,84 @@ critical_check_result=0
 
 # machine check
 # expected values are from step `ipi-conf-azure-confidential-trustedlaunch`
-if [[ "${ENABLE_TRUSTEDLAUNCH_DEFAULT_MACHINE}" == "true" ]]; then
+master_encryptionathost=""
+master_security_type=""
+master_secure_boot=""
+master_vtpm=""
+master_osdisk_securityencryptiontype=""
+worker_encryptionathost=""
+worker_security_type=""
+worker_secure_boot=""
+worker_vtpm=""
+worker_osdisk_securityencryptiontype=""
+if [[ "${ENABLE_CONFIDENTIAL_DEFAULT_MACHINE}" == "true" ]]; then
     master_encryptionathost=$(yq-go r "${INSTALL_CONFIG}" 'platform.azure.defaultMachinePlatform.encryptionAtHost')
     master_security_type=$(yq-go r "${INSTALL_CONFIG}" 'platform.azure.defaultMachinePlatform.settings.securityType')
-    master_secure_boot=$(yq-go r "${INSTALL_CONFIG}" 'platform.azure.defaultMachinePlatform.settings.trustedLaunch.uefiSettings.secureBoot')
-    master_vtpm=$(yq-go r "${INSTALL_CONFIG}" 'platform.azure.defaultMachinePlatform.settings.trustedLaunch.uefiSettings.virtualizedTrustedPlatformModule')
+    master_secure_boot=$(yq-go r "${INSTALL_CONFIG}" "platform.azure.defaultMachinePlatform.settings.${master_security_type,}.uefiSettings.secureBoot")
+    master_vtpm=$(yq-go r "${INSTALL_CONFIG}" "platform.azure.defaultMachinePlatform.settings.${master_security_type,}.uefiSettings.virtualizedTrustedPlatformModule")
     worker_encryptionathost="${master_encryptionathost}"
     worker_security_type="${master_security_type}"
     worker_secure_boot="${master_secure_boot}"
     worker_vtpm="${master_vtpm}"
+    if [[ -n "${OS_DISK_SECURITY_ENCRYPTION_TYPE}" ]]; then
```

Comment:
I guess we do not need the if condition, if there is no `osDisk.securityProfile.securityEncryptionType` setting, the following yq command can return an empty value.

Reply:
updated, then ENV `OS_DISK_SECURITY_ENCRYPTION_TYPE` is not required any more, removed it.